Add workaround on securityContext.runAsUser error (#4368)
operator-framework/operator-sdk: website/content/en/docs/faqs/_index.md

Now run `make manifests` to update your `role.yaml`.

## I keep seeing errors like `securityContext.runAsUser: Invalid value: 65532: must be in the ranges: [<NNN>, <MMM>]]`
	
Note that, the error can only be checked for some specific cluster's vendors such as OCP. In order to fix the issue,  replace `runAsUser: 65532` with `runAsNonRoot: true` in the `config/manager/manager.yaml` file :

```yaml
spec:
      securityContext:

> +        runAsNonRoot: true 

Author: ```suggestion
        runAsNonRoot: true 
```

Now run `make manifests` to update your `role.yaml`.

> ## Seeing errors about `securityContext.runAsUser: Invalid value: 65532: must be in the ranges: [<NNN>, <MMM>]]`

Reviewer: ```suggestion
## I keep seeing errors like `securityContext.runAsUser: Invalid value: 65532: must be in the ranges: [<NNN>, <MMM>]]`
```

> For instance, if you are creating the `--kind=memcached` operator with `operator-sdk`. A workaround for this specific instance is to change the generated `config/manager/manager.yaml`:

Reviewer: ```suggestion
Note that, the error can only be checked for some specific cluster's vendors such as OCP. In order to fix the issue,  replace `runAsUser: 65532` with `runAsNonRoot: true` in the `config/manager/manager.yaml` file :
```

Reviewer: It is not specific for who are creating an API are creating the `--kind=memcached` operator with `operator-sdk`. Indeed the root cause is because of the vendor. Then, because of this, I do not think that we ought to say that.

> ```diff

Reviewer: ```suggestion
```yaml
```

Author: I think diff format is better than yaml here as it quickly shows what to remove and what to add in a commonly recognized format.

Author: ah okay, didn't realize the doc philosophy

spec:
      securityContext:

> -        runAsUser: 65532

Reviewer: ```suggestion
```

Author: I think diff format is better than yaml here as it quickly shows what to remove and what to add in a commonly recognized format.

Reviewer: I think it differs from the current layout of the docs. We try to make thee things easier for users are able to follow up the steps by copying and pasting them which would not be possible with this approach.
